examples/gcp_flow_logs_pubsub_topic/versions.tf

```
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 4.0"
```

Did you copy-and-paste this from the Azure submodules?
I doubt `4.0` would work.
The current version is `7.14.0`:
https://registry.terraform.io/providers/hashicorp/google/latest/docs
Use that one, if that's the version you have tested with.
```suggestion
      version = ">= 7.14.0"
```